Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4029,3 +4047,48 @@ def update_integration_context(context, object_keys=None, sync=True):
 
 class DemistoException(Exception):
     pass
+
+
+class ObjectMirror(object):
```

Comment:
I think maybe to change the name of the class to `GetRemoteDataResponse`

Reply:
Similar class we need for inputs.
`parse_get_remote_data_args(args) -> GetRemoteDataArgs`
`GetRemoteDataArgs`

`parse_update_remote_system_args(args) -> UpdateRemoteSystemArgs`
`UpdateRemoteSystemArgs`
and also `UpdateRemoteSystemResponse`